crates/re_space_view_spatial/src/scene/contexts/transform_context.rs

```diff
@@ -15,7 +15,7 @@ struct TransformInfo {
     ///
     /// None indicates that this entity is under the eye camera with no Pinhole camera in-between.
     /// Some indicates that the entity is under a pinhole camera at the given entity path that is not at the root of the space view.
-    pub parent_pinhole: Option<EntityPathHash>,
+    pub parent_pinhole: Option<EntityPath>,
```

Comment:
Out of curiosity, why drop the usage of Hash here? Doesn't this strictly introduce more Arc-related reference-counting overhead?

Reply:
Was not happy about that either, but that allowed me to re-use that on the depth cloud setup code which previously had its own logic for the same thing. That piece however, wants to know the parent of the pinhole (hmm maybe that's an issue actually), so it needed the whole path, not just the hash.